### ISSUE TITLE
Feature Request: Show Tokens Above

### ISSUE THREAD

**Issue by TheGreatRedDuke** [theripper93/Levels#20]
It would be great to have a way to "Show tokens above" that would always render tokens on higher levels. This way, you could have flying tokens that are always above any other levels.

For example, I have a map that has multiple overhead elements that the player tokens walk below. I can add a second level tile and set it to "Show even when below?" which gives me the great overhead effect.  But then there's no way to have the flying enemy token appear above the overhead tile and be visible to the players; it's always partially stuck under that 2nd level overhead tile as well. (screenshots added to show this in action)

Maybe what I'm trying to do can already be done, but I can't seem to figure it out.

Anyway, great work on these modules. My troubles aside, they are really great additions to the Foundry platform!

![Screenshot 2021-06-24 233906](https://user-images.githubusercontent.com/78890368/123366131-a73a2f00-d545-11eb-9233-9d299f112222.png)

![Screenshot 2021-06-24 234000](https://user-images.githubusercontent.com/78890368/123366132-a7d2c580-d545-11eb-9a2a-786115e53efa.png)

**Comment by theripper93** [comment]
This can 100% be done already, set a brmode to the tile (would suggest show) , and enable advanced sight

**Comment by TheGreatRedDuke** [comment]
Ahh, that's great to hear! But sorry for a dumb question... what is
'advanced sight'? I've looked for a module called that (or something like
it) and I've looked through all the Levels and Better Roofs module settings
and config panels and can't find any settings by that name, nor anything in
the Foundry settings or when I edit the tile settings. This must be the
piece I was missing in my testing.
